clients/RemoteDisplay/input_receiver.c

```diff
@@ -565,22 +566,20 @@ init_transport(struct input_receiver_private_data *data)
 	}
 
 	transport->input.addr.sin_family = AF_INET;
-	transport->input.addr.sin_addr.s_addr = inet_addr(transport->str_ipaddr);
+	transport->input.addr.sin_addr.s_addr = htonl(INADDR_ANY);
 	transport->input.addr.sin_port = htons(transport->input.port);
 
-	/* Send a simple hello message to the server so that it knows about us */
-	ret = sendto(transport->input.sock_desc, (const char *)hello, strlen(hello),
-			0, (const struct sockaddr *) &transport->input.addr,
-			transport->input.len);
-
+	ret = bind (transport->input.sock_desc, (struct sockaddr *) &transport->input.addr,
```

Comment:
minor nitpick but sometimes after the function name you put a space (ex: bind above) and sometimes you don't (ex: htonl above). We usually don't put a space for IAS code. I will remove the spaces myself and check it in, but this is something to remember for next time.